[INFO   ] Logger: Record log in /storage/emulated/0/kivy/03_07_2021/.kivy/logs/kivy_21-07-03_38.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, May 27 2020, 02:08:17) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/kivy/03_07_2021/main.py", line 4, in <module>
[WARNING] stderr:     from functiools import partial
[WARNING] stderr: ModuleNotFoundError: No module named 'functiools'
